.eslintrc

```
@@ -0,0 +1,36 @@
+// this file was auto-generated, do not edit it directly.
+// instead run bin/update_build_scripts from
+// https://github.com/sharelatex/sharelatex-dev-environment
+// Version: 1.1.21
+{
+  "extends": [
+    "standard",
+    "prettier",
+    "prettier/standard",
+  ],
+  "plugins": [
+    "mocha",
+    "chai-expect",
+    "chai-friendly"
+  ],
+  "env": {
+    "node": true
```

This should prevent failing due `no-error` and maybe other lint rules applied to unit tests.

```suggestion
    "node": true,
    "mocha": true
```

---

Good catch.

Since this file is auto-generated, and the mocha environment only applies to tests, I'll add that to `test/.eslintrc.json`

The added `expect` global should not be necessary. It needs to be imported in tests. According to Ali, it's there in web because we place it in the global scope somewhere.